docs/src/pages/[platform]/components/alert/examples/AlertThemeExample.tsx

```diff
@@ -0,0 +1,47 @@
+import { Alert, Flex, ThemeProvider, Theme } from '@aws-amplify/ui-react';
+
+const theme: Theme = {
+  name: 'alert-theme',
+  tokens: {
+    components: {
+      alert: {
+        // Default styles
+        backgroundColor: { value: '{colors.teal.20}' },
+
+        icon: {
+          size: { value: '{fontSizes.xxxl}' },
+        },
+
+        heading: {
+          fontSize: { value: '{fontSizes.large}' },
+          fontWeight: { value: '{fontWeights.normal}' },
+        },
+
+        // Variations
+        info: {
+          color: { value: 'white' },
+          backgroundColor: { value: '{colors.purple.80}' },
```

Comment:
Would you mind replacing the uses of purple in the examples with blue? I think the purple clashes with our other default theme colors.

Reply:
Yep